bibtex.go

```diff
@@ -11,6 +11,22 @@ import (
 	"time"
 )
 
+//prettyPrintDefaultKeyOrder controls the order in which BibEntry keys are printed
+//when using the PrettyString() instead of the PrettyStringCustom(keyOrder []string) variants.
+//Index 0 becomes first field and so on.
```

Comment:
Can we make sure it's consistent with rest of the code base, with a space between `//` and comment
```suggestion
// prettyPrintDefaultKeyOrder controls the order in which BibEntry keys are printed
// when using the PrettyString() instead of the PrettyStringCustom(keyOrder []string) variants.
// Index 0 becomes first field and so on.
```